internal/connectors/httpwrapper/config.go

```diff
@@ -2,15 +2,29 @@ package httpwrapper
 
 import (
 	"net/http"
+	"os"
 	"time"
 
+	"go.opentelemetry.io/otel/attribute"
 	"golang.org/x/oauth2/clientcredentials"
 )
 
 type Config struct {
-	HttpErrorCheckerFn func(code int) error
+	HttpErrorCheckerFn      func(code int) error
+	CommonMetricsAttributes []attribute.KeyValue
 
 	Timeout     time.Duration
 	Transport   http.RoundTripper
 	OAuthConfig *clientcredentials.Config
 }
+
+func CommonMetricsAttributesFor(connectorName string) []attribute.KeyValue {
+	metricsAttributes := []attribute.KeyValue{
+		attribute.String("connector", connectorName),
+	}
+	stack := os.Getenv("STACK")
+	if stack != "" {
+		metricsAttributes = append(metricsAttributes, attribute.String("stack", stack))
```

Comment:
I'm wondering if the stack is not yet added by the exporter or other services handling metrics, let's ask Max

Reply:
I was just imitating what was being done in payments v2 but looking at the dev cluster it looks like it would normally be exported to OTEL_RESOURCE_ATTRIBUTES:

```
OTEL_METRICS:                         true                                                                                                                                              │
│       OTEL_METRICS_BATCH:                   true                                                                                                                                              │
│       OTEL_METRICS_EXPORTER:                otlp                                                                                                                                              │
│       OTEL_METRICS_EXPORTER_OTLP_INSECURE:  true                                                                                                                                              │
│       OTEL_METRICS_EXPORTER_OTLP_MODE:      grpc                                                                                                                                              │
│       OTEL_METRICS_PORT:                    4317                                                                                                                                              │
│       OTEL_METRICS_ENDPOINT:                otel-collector-svc.monitoring.svc.cluster.local                                                                                                   │
│       OTEL_METRICS_EXPORTER_OTLP_ENDPOINT:  $(OTEL_METRICS_ENDPOINT):$(OTEL_METRICS_PORT)                                                                                                     │
│       OTEL_SERVICE_NAME:                    payments                                                                                                                                          │
│       OTEL_RESOURCE_ATTRIBUTES:             stack=unbhznflobkp-mixp                                                                                                                           │
│       STACK_URL:                            http://gateway:8080                                                                                                                               │
│       STACK_PUBLIC_URL:                     https://unbhznflobkp-mixp.vadztti.formance.dev                                                                                                    │
│       DEBUG:                                false                                                                                                                                             │
│       DEV:                                  false                                                                                                                                             │
│       STACK:                                unbhznflobkp-mixp
```

I'll double check what go-libs does in this setup